templates/.github/workflows/publish-to-pypi.yml

```diff
@@ -0,0 +1,72 @@
+name: PyPI
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build-n-publish:
+
+    strategy:
+      matrix:
+        python-version: [ '2.x', '3.6', '3.7', '3.8', '3.9', 'pypy-2.7', 'pypy-3.6', 'pypy-3.7' ]
+        os: [ubuntu-latest]
```

Comment:
Overall LGTM! This line might feel redundant, but I guess it's because you want to add other platforms in the future. All good!